ops/lib/__init__.py

```diff
@@ -113,7 +113,7 @@ def _find_all_specs(path):
             if finder is None or not hasattr(finder, 'find_spec'):
                 continue
             for lib_dir in lib_dirs:
-                spec = finder.find_spec(lib_dir)
+                spec = finder.find_spec("{}.opslib.{}".format(top_dir, lib_dir))
```

Comment:
I'm glad that this works, but I don't understand why it does.

Reply:
say you have a `pgsql/opslib/devel/__init__.py` module that says it's `LIBNAME=pgsql` etc such that you can do
```python
pgsql = ops.lib.use('pgsql', 2, 'alice@example.com')
```

the 'regular' way to import the module would be, e.g.,
```python
from pgsql.opslib import devel as pgsql
```

If a module uses relative imports, it's that module's full name that is then used to resolve them, so when you do `from . import foo` it knows to get `pgsql/opslib/devel/foo.py`. But if the 'full' name of the module does not have those other bits, it gets very confused (it loses track of the module's 'location', which is what pops out in the import error)